frontend/amp/components/elements/TextBlockComponent.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { css } from 'react-emotion';
+import { palette } from '@guardian/pasteup/palette';
+import { pillarPalette } from '../../../pillars';
+
+// tslint:disable:react-no-dangerous-html
+const style = (pillar: Pillar) => css`
+    p {
```

Comment:
Presumably we'll have to add ul and other element styling later?

Reply:
Yeah, this was an oversight.